public/index.php

```diff
@@ -38,6 +38,8 @@
 if (!in_array($badgeStyle, ['flat', 'flat-square', 'plastic', 'for-the-badge', 'pixel'], true)) {
     $badgeStyle = 'flat';
 }
+$baseNumber = $request->baseNumber() ?? '0';
+$baseNumber = ctype_digit($baseNumber) ? intval($baseNumber) : 0;
```

Comment:
Silently sets the base number to 0 if an invalid (negative or non-digit) value is provided.
For extremely large values, `intval` will return `9223372036854775807`

Reply:
I'd better render an error on non digit or too large value